src/runtime/plugins/date.ts

```diff
@@ -3,8 +3,8 @@ import type { VuetifyOptions } from 'vuetify'
 import { useNuxtApp } from '#imports'
 
 export function configureDate(vuetifyOptions: VuetifyOptions) {
-  if (adapter === 'custom' || !enabled)
-    return
+  // if (adapter === 'custom' || !enabled)
+  //   return
 
```

Comment:
shouldn't be 
```js
if (!enabled)
    return
```
?

Reply:
i just know when vuetifyOptions.date.adapter is NOT set to 'custom',
configureDate plugin won't change date localization.
I assume your code is right 👍👍👍

Reply:
@mmnoori can you explain the problem? changing the code with my previous suggestion will fix the problem you're trying to solve?